ynr/apps/moderation_queue/review_required_helper.py

```diff
@@ -0,0 +1,150 @@
+import abc
+from collections import namedtuple
+
+from django.conf import settings
+
+
+class BaseReviewRequiredDecider(metaclass=abc.ABCMeta):
+    """
+    A base class that decides if a given LoggedAction needs to be flagged
+    as requiring review
+    """
+
+    def __init__(self, logged_action):
+        """
+        :type logged_action: candidates.models.LoggedAction
+        """
+        self.logged_action = logged_action
+
+    @abc.abstractmethod
+    def review_description_text(self):
```

Comment:
It would be nicer to make these two methods `@property` given we don't need to pass them params.